docker/Dockerfile.py

```diff
@@ -117,7 +117,7 @@ def heredoc(s):
 
     # This component changes most frequently and keeping it last maximizes Docker cache hits.
     COPY {sdistName} .
-    RUN pip install {sdistName}[aws,mesos,encryption,cwl]
+    RUN pip install {sdistName}[aws,mesos,encryption,cwl,azure]
```

Comment:
Should this be `pip install {sdistName}[all]` now that we have #2029?

Reply:
No. We don't HTCondor.